### PR TITLE
fix: not trigger authState update during login redirect

### PR DESCRIPTION
**Pull request by shuowu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@okta/okta-auth-js",
   "description": "The Okta Auth SDK",
-  "version": "5.4.1",
+  "version": "5.4.2",
```
Comment (aarongranick-okta) [inline package.json:5]: Why is version 5.4.1 in master? For a patch release this PR should be against 5.4 branch

Comment (aarongranick-okta) [reply package.json:5]: Looks like the branch may be based on 5.4 but the PR is based on master. please fix.

Comment (shuowu) [reply package.json:5]: good catch, actually I don't know why... 
This PR looks correct  https://github.com/okta/okta-auth-js/pull/919/files

Comment (shuowu) [reply package.json:5]: OK, just the PR issue, changed.

Comment (aarongranick-okta) [reply package.json:5]: It is correct in master: https://github.com/okta/okta-auth-js/blob/master/package.json#L5

**Review by aarongranick-okta — APPROVED** [review]

**Pull request closed without merging**